app/views/layouts/application.html.erb

```diff
@@ -42,8 +42,13 @@
       document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
     </script>
 
+    <div class="app-cookie-banner" data-module="cookie-message">
+      <p class="app-cookie-banner__message">GOV.UK uses cookies to make the site simpler. <%= govuk_link_to "Find out more about cookies", provider_interface_cookies_path %></p>
```

Comment:
There's a `candidate_interface_cookies_path` too, maybe this needs something like what we're doing with `<% case current_namespace %>` further down in this file?

Reply:
Ah  yes, should we use `candidate_interface_cookies_path ` for now then?

Reply:
Can we add the `case` or `if/else`? 🙏